tests/shopify_dlt/test_shopify_pipeline.py

```diff
@@ -0,0 +1,20 @@
+from tests.utils import ALL_DESTINATIONS, assert_load_info, load_table_counts
+import pytest
+import dlt
+from pipelines.shopify_dlt import shopify_source
+
+@pytest.mark.parametrize('destination_name', ALL_DESTINATIONS)
+def test_all_resources(destination_name: str) -> None:
+    pipeline = dlt.pipeline(pipeline_name='hubspot', destination=destination_name, dataset_name='asana_data', full_refresh=True)
+    load_info = pipeline.run(shopify_source())
+    print(load_info)
+    assert_load_info(load_info)
+    table_names = [t["name"] for t in pipeline.default_schema.data_tables()]
+    table_counts = load_table_counts(pipeline, *table_names)
+
+    # for now only check main tables
+    expected_tables = ['products', 'orders', 'customers']
+    assert set(table_counts.keys()) > set(expected_tables)
+    assert table_counts['products'] == 17
+    assert table_counts['orders'] == 10
+    assert table_counts['customers'] == 3
```

Comment:
load again to make sure incremental loading works (counts should not change)

Reply:
done (I think)